prov/gni/include/gnix_vc.h

```diff
@@ -107,10 +111,16 @@ enum gnix_vc_conn_req_type {
  *                           the VC not pertaining to the connection state.
  */
 struct gnix_vc {
```

Comment:
looks like the doxygen comments are getting out of sync with the struct.  would you mind updating?